Client/Frontend/Browser/Onboarding/Welcome/ReleaseNotesView.swift

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveShared
+
+struct ReleaseNotesView: View {
+    @Environment(\.colorScheme) var colorScheme: ColorScheme
+
+    var dismiss: (() -> Void)?
+
+    var body: some View {
+        VStack {
+            Image(uiImage: #imageLiteral(resourceName: "brave-vpn-ad-top-shimmer"))
+            .resizable()
+            .aspectRatio(contentMode: .fit)
+            HStack {
+                Spacer()
+                Button {
+                    dismiss?()
+                } label: {
+                    Image(uiImage: #imageLiteral(resourceName: "privacy-everywhere-exit-icon"))
+                }
+                .padding(.trailing, 10)
+                .padding(.top, 10)
+            }
+            Text("Brave new releases")
+            Image(uiImage: #imageLiteral(resourceName: "brave-vpn-ad-background"))
+            .resizable()
+            .aspectRatio(contentMode: .fit)
+            .overlay(imageOverlay, alignment: .center)
+            .padding()
+            VStack(alignment: .leading, spacing: 8.0) {
+                Text("• One line of release notes")
+                Text("• Another line of release notes")
+            }
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .padding()
+        }
+        .frame(maxWidth: 450)
+        .background(Color(.braveBackground))
+        .cornerRadius(5.0)
+    }
+    
+    private var imageOverlay: some View {
+        VStack {
+            HStack {
+                Text("Firewall")
+                    .font(Font(UIFont.systemFont(ofSize: 27.0, weight: .bold)))
+                    .rotationEffect(Angle(degrees: -9.57), anchor: .bottomLeading)
+                    .foregroundColor(Color.white)
+                
+                Text("+")
+                    .font(Font(UIFont.systemFont(ofSize: 35.0, weight: .bold)))
+                    .foregroundColor(Color.white)
+                    .offset(y: -15.0)
+            }.offset(x: -20.0, y: 10.0)
+            
+            Text("VPN")
+                .foregroundColor(Color.white)
+                .font(Font(UIFont.systemFont(ofSize: 56.0, weight: .bold)))
+                .padding()
+                .overlay(RoundedRectangle(cornerRadius: 12.0)
+                            .stroke(Color.white, lineWidth: 9.5)
+                )
+                .offset(y: -15.0)
+        }
+    }
```

Comment:
- Don't use UIFont's, don't use static font sizes
- Don't use `offset`, adjust stack spacing
- Missing localization.

If this is supposed to be replacing an image, best to honestly just swap it with an actual image and not recreate it like this

Reply:
Same as https://github.com/brave/brave-ios/pull/4508#discussion_r750624936